### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by Adri2000**

### CODE REVIEW

File: user/pages/03.Howtos/05.nova-rescue-mode/docs.en.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ $ sudo testdisk /dev/vdb1
 
 TestDisk 7.0, Data Recovery Utility, April 2015
 Christophe GRENIER <grenier@cgsecurity.org>
-http://www.cgsecurity.org
+https://www.cgsecurity.org
```
Comment (matthiasbernhardt) [inline user/pages/03.Howtos/05.nova-rescue-mode/docs.en.md:151]: Changing the quoted output of a 3rd party tool goes a bit far, don't you think?

Comment (Adri2000) [reply user/pages/03.Howtos/05.nova-rescue-mode/docs.en.md:151]: Ahah, true, I missed that!

File: user/pages/04.Reference/03.compute/docs.en.md [hunk 2]
```diff
@@ -268,5 +268,5 @@ The security group "default" is added in this example, since this group is takin
 Any occupied resources are considered "used" and counted against your [quota limits](../../05.Background/03.resource-limits/docs.en.md#usage). They will also be counted against your upfront commitment or charged on demand.
 When you create an instance, volume, snapshot, object, loadbalancer, floating ip, dns zone, resources will be allocated and so they will be counted and charged, no matter if they are actually running, attached, assigned.
 To save money or free quota, it is thus not sufficient to stop, detach or unassign them, you must delete them completely to free the resources so that we can reuse them.
-It is part of higher level automation like kubernetes or terraform or similar tools to recreate them if needed.
+It is part of higher level automation like Kubernetes or terraform or similar tools to recreate them if needed.
```
Comment (matthiasbernhardt) [inline user/pages/04.Reference/03.compute/docs.en.md:271]: Either `kubernetes or terraform` or `Kubernetes or Terraform` but not mixed, please.

File: user/pages/04.Reference/05.object-storage/docs.en.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ Basically, SysEleven Stack Object Storage is a big key/value store.
 A file or file object can be assigned a file name like key, and made available under this key.
 
 !! **Be aware**
-!! We discourage the use of special characters, especially dots (`.`) and slashes (`/`) in bucket or object names, especially at the start and end of names. As names can be used or interpreted as dns names, pathnames and/or filenames, this can confuse both server and client software and in consequence may lead to buckets or objects being unaccessible or unmaintainable. In case you are stuck with such a phenomenon, please contact our [Support (support@syseleven.de)](../../06.Support/default.en.md).
+!! We discourage the use of special characters, especially dots (`.`) and slashes (`/`) in bucket or object names, especially at the start and end of names. As names can be used or interpreted as dns names, path names and/or filenames, this can confuse both server and client software and in consequence may lead to buckets or objects being unaccessible or unmaintainable. In case you are stuck with such a phenomenon, please contact our [Support (support@syseleven.de)](../../06.Support/default.en.md).
```
Comment (matthiasbernhardt) [inline user/pages/04.Reference/05.object-storage/docs.en.md:56]: Either `pathnames and/or filenames` or `path names and/or file names` but not mixed, please. Actually, I prefer to not change anything here but if, then both.

**Review by matthiasbernhardt — APPROVED** [review]
lgtm

**Pull request merged**